models/renalStructures_UNEST_segmentation/configs/inference.json

```diff
@@ -112,7 +112,7 @@
             "_target_": "CheckpointLoader",
             "load_path": "$@bundle_root + '/models/model.pt'",
             "load_dict": {
-                "state_dict": "@network"
+                "model": "@network"
```

Comment:
If you update the bundle content, please update the version in the `metadata.json`.

Thanks.

Reply:
Done

Reply:
I didn't see you updated the version in the `metadata.json`, did you push the commit?

Thanks.

Reply:
```
    "changelog": {
        "0.1.0": "complete the model package",
        "0.1.1": "initialize the model package structure",
        "0.1.2": "Fixed the model state_dict name"
    },
```

It should be this section right? 
Yep, committed, thanks!